app/assets/stylesheets/components/links.scss

```diff
@@ -0,0 +1,83 @@
+////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+// .c-link /////////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+
+.c-link {
+  position: relative;
+  display: inline-block;
+  border-radius: var(--radius);
+  font: inherit;
+  cursor: pointer;
+  transition: all var(--transition-props);
+  outline: 0;
+  text-decoration: none;
+  color: var(--color);
+
+  &--block {
+    display: flex;
+    padding: var(--su-2) var(--su-4);
+    overflow-wrap: normal;
+    background-color: var(--bg);
+
+    &:hover {
+      background-color: var(--bg-hover);
+    }
+  }
+
+  &:hover {
+    color: var(--color-hover);
+    z-index: var(--z-elevate);
+    text-decoration: underline;
```

Comment:
I think we should also add this text-decoration on focus

Reply:
good catch, adding!